From 199731cd7ef269926a5235ca4efe48966db14dc1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E5=87=AF?= <huangkai@hosonsoft.com>
Date: Sat, 18 Apr 2026 17:28:56 +0800
Subject: [PATCH 3/3] =?UTF-8?q?=E4=BF=AE=E6=94=B9=20cef=20=E7=89=88?=
 =?UTF-8?q?=E6=9C=AC=E5=8F=B7?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 include/CefVersion.h | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/include/CefVersion.h b/include/CefVersion.h
index 1f20f07..da85c65 100644
--- a/include/CefVersion.h
+++ b/include/CefVersion.h
@@ -13,12 +13,12 @@
 #pragma once
 
 // clang-format off
-#define CEF_VERSION "142.0.15+g6dfdb28+chromium-142.0.7444.176"
-#define CEF_VERSION_MAJOR 142
-#define CEF_VERSION_MINOR 0
-#define CEF_VERSION_PATCH 15
-#define CEF_COMMIT_NUMBER 3314
-#define CEF_COMMIT_HASH "6dfdb28d752a47e189d7a23b01f368ab0bdb378d"
+#define CEF_VERSION "127.3.5+g114ea2a+chromium-127.0.6533.120"
+#define CEF_VERSION_MAJOR 127
+#define CEF_VERSION_MINOR 3
+#define CEF_VERSION_PATCH 5
+#define CEF_COMMIT_NUMBER 3037
+#define CEF_COMMIT_HASH "114ea2af1ba9da18c4ac5e599ccdbb17d01ba75a"
 // clang-format on
 
 #endif // CefVersion
-- 
2.22.0.windows.1

